weather-app/src/app/favorite/favorite.component.html

```diff
@@ -0,0 +1,12 @@
+<button class="favorite__btn main-container">Favorites:</button>
+
+<div class="favorite main-container">
```

Comment:
change favorite to favorite-list

Reply:
and not necessary to add class "main-container" because there is already in main